### PR TITLE
Update notebooks to run on Watson Studio and add legal forms

### PR DESCRIPTION
**Pull request by tlzhu19**
This PR covers the following stories:

- Split notebooks into two (#602)
- Add data visualization (#603)
- Update documentation and test in Watson Studio (#604)
- Update legal forms (#605)

### CODE REVIEW

**Comment by yil532** [comment]
Left comments in the slack channel.

**Comment by tlzhu19** [comment]
<p>thanks, updated the PR with the fixes</p>

File: notebooks/Part 1 - Data Exploration & Visualization.ipynb [hunk 1]
```diff
@@ -0,0 +1 @@
...AI Technologies](http://codait.org).\n\nCopyright \u00a9 2020 IBM. This notebook and its source code are released under the terms of the MIT License."}, {"metadata": {}, "cell_type": "markdown", "source": "<div style=\"background:#F5F7FA; height:110px; padding: 2em; font-size:14px;\">\n<span style=\"font-size:18px;color:#152935;\">Love this notebook? </span>\n<span style=\"font-size:15px;color:#152935;float:right;margin-right:40px;\">Don't have an account yet?</span><br>\n<span style=\"color:#5A6872;\">Share it with your colleagues and help them discover the power of Watson Studio!</span>\n<span style=\"border: 1px solid #3d70b2;padding:8px;float:right;margin-right:40px; color:#3d70b2;\"><a href=\"https://ibm.co/wsnotebooks\" target=\"_blank\" style=\"color: #3d70b2;text-decoration: none;\">Sign Up</a></span><br>\n</div>"}], "metadata": {"kernelspec": {"name": "python3", "display_name": "Python 3.7", "language": "python"}, "language_info": {"name": "python", "version": "3.7.9", "mimetype": "text/x-python", "codemirror_mode": {"name": "ipython", "version": 3}, "pygments_lexer": "ipython3", "nbconvert_exporter": "python", "file_extension": ".py"}}, "nbformat": 4, "nbformat_minor": 4}
```
Comment (yil532) [inline notebooks/Part 1 - Data Exploration & Visualization.ipynb]: <p>Let's leave the bucket_name as </p><pre class="ql-syntax" spellcheck="false">'bucket_name': 'ibmdebaterthematicclusteringofsen.....'
</pre><p>Also, asset_id as</p><pre class="ql-syntax" spellcheck="false">'asset_id': '...'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-thematic-clustering-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (yil532) [inline notebooks/Part 1 - Data Exploration & Visualization.ipynb]: <p>Change it to 2021 IBM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-thematic-clustering-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

File: notebooks/Part 2 - Model Development.ipynb [hunk 2]
```diff
@@ -0,0 +1 @@
...AI Technologies](http://codait.org).\n\nCopyright \u00a9 2020 IBM. This notebook and its source code are released under the terms of the MIT License."}, {"metadata": {}, "cell_type": "markdown", "source": "<div style=\"background:#F5F7FA; height:110px; padding: 2em; font-size:14px;\">\n<span style=\"font-size:18px;color:#152935;\">Love this notebook? </span>\n<span style=\"font-size:15px;color:#152935;float:right;margin-right:40px;\">Don't have an account yet?</span><br>\n<span style=\"color:#5A6872;\">Share it with your colleagues and help them discover the power of Watson Studio!</span>\n<span style=\"border: 1px solid #3d70b2;padding:8px;float:right;margin-right:40px; color:#3d70b2;\"><a href=\"https://ibm.co/wsnotebooks\" target=\"_blank\" style=\"color: #3d70b2;text-decoration: none;\">Sign Up</a></span><br>\n</div>"}], "metadata": {"kernelspec": {"name": "python3", "display_name": "Python 3.7", "language": "python"}, "language_info": {"name": "python", "version": "3.7.9", "mimetype": "text/x-python", "codemirror_mode": {"name": "ipython", "version": 3}, "pygments_lexer": "ipython3", "nbconvert_exporter": "python", "file_extension": ".py"}}, "nbformat": 4, "nbformat_minor": 4}
```
Comment (yil532) [inline notebooks/Part 2 - Model Development.ipynb]: <p>2021 IBM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-thematic-clustering-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by tlzhu19** [comment]
@yil532 updated with your comments

**Review by yil532 — APPROVED** [review]
LGTM

**Pull request merged**